types/+laxMPC/code_laxMPC_ADMM_C.c

```diff
@@ -722,20 +687,14 @@ void laxMPC_ADMM(double *x0_in, double *xr_in, double *ur_in, double *u_opt, int
 
     // Measure time
     #if MEASURE_TIME == 1
-    
-    #if WIN32
-    QueryPerformanceCounter(&post_polish); // Get time after polishing
-    t_run_time = 1000000000ULL * (post_polish.QuadPart - start.QuadPart) / frequency.QuadPart;
-    t_polish_time = 1000000000ULL * (post_polish.QuadPart - post_solve.QuadPart) / frequency.QuadPart;
-    sol->run_time = t_run_time/(double)1e+9;
-    sol->polish_time = t_polish_time/(double)1e+9;
-    #else // If Linux
-    clock_gettime(CLOCK_MONOTONIC_RAW, &post_polish);
-    sol->run_time = (double) ( (post_polish.tv_sec - start.tv_sec) * 1000.0 ) + (double) ( (post_polish.tv_nsec - start.tv_nsec) / 1000000.0 );
-    sol->polish_time = (double) ( (post_polish.tv_sec - post_solve.tv_sec) * 1000.0 ) + (double) ( (post_polish.tv_nsec - post_solve.tv_nsec) / 1000000.0 );
-    #endif
-    
+    read_time(&post_polish);
+    get_elapsed_time(&sol->polish_time, &post_polish, &post_solve);
+    get_elapsed_time(&sol->run_time, &post_polish, &start);
     #endif
 
 }
 
+spcies_function_read_elapsed_time();
```

Comment:
This is correct, although I have changed it so that now the format is `spcies_snippet_name();`, because the feature does not only have to be for functions, but for inserting code (or snippets) in general. Also, the snippets are now located in `spcies_root/snippets/`, instead of `spcies_root/type/functions/`. I think this location is more intuitive.